crates/trycp_server/src/reset.rs

```diff
@@ -58,5 +41,10 @@ pub(crate) fn reset() -> Result<(), ResetError> {
         }
     }
 
+    NEXT_ADMIN_PORT.store(ADMIN_PORT_RANGE.start, atomic::Ordering::SeqCst);
+    if let Err(err) = std::fs::remove_dir_all(PLAYERS_DIR_PATH) {
```

Comment:
In very few cases this directory is already gone. That shouldn't make the whole call fail.